src/components/ImportUUIDs/ImportUUIDs.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { noop } from 'lodash';
+
+import { FileUploader } from '@folio/stripes-data-transfer-components';
+import { Layout } from '@folio/stripes-components';
+
+import css from './ImportUUIDs.css';
+
+export function ImportUUIDs() {
```

Comment:
As far as I understand we should have ability to upload files with UIIDs and CQL queries. Are we going to have the separated component to upload CQL query or it will be handled in scope of this one?

Reply:
Good point, I think it will be in the scope of this one. Let's come up with the proper name to fulfill both. I am thinking about `ImportRecords` name. What do you think?

Reply:
I think I need a little bit more information to understand your future steps to select a component name. But as far as I understand this component is just responsible for a file uploading: loads file and send it to backend; and it do nothing to import records. Do I misunderstand your idea?

Reply:
I plan to have the logic related to loading files in this component. So this component will handle ids and query.

Reply:
I understand it, but it seems you are not going to import records here.

Reply:
Actually, I expect to do it here.

Reply:
Agreed to name it QueryFileUploader.